rules_building_block/discovery_signal_unusual_user_host.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2023/10/10"
+maturity = "production"
+min_stack_comments = "Multiple field support in the New Terms rule type was added in Elastic 8.6"
+min_stack_version = "8.6.0"
+updated_date = "2023/10/10"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule leverages alert data from various Discovery building block rules to alert on signals with unusual unique
+host.id and user.id entries.
+"""
+from = "now-9m"
+index = [".alerts-security.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Unusual Discovery Signal Alert with Unusual Host/User ID Combination"
```

```suggestion
name = "Unusual Discovery Activity by User"
```
Maybe we could be more generic here and in the description